src/manifest.json

```diff
@@ -25,6 +25,13 @@
   },
   "options_page": "pages/options/index.html",
   "content_scripts": [
+    {
+      "matches": ["*://*/*"],
+      "run_at": "document_start",
+      "js": [
+        "content_scripts/prevent-indexeddb-tracking/index.js"
```

Comment:
this is intended only for Safari

Reply:
After all we need separate manifests for each target, so I introduced suffixed version of manifest, and this content script is only loaded on Safari.